src/webots/nodes/WbLidar.cpp

```diff
@@ -679,7 +681,10 @@ void WbLidar::updateType() {
     parsingWarn(tr("'type' should either be 'fixed' or 'rotating', reset to 'fixed'"));
     mType->setValue("fixed");
   }
-  if (areWrenObjectsInitialized())
+  if (hasBeenSetup())
+    warn(tr("'type' has been modified. This modification will be taken into account after saving and reloading "
+            "the world."));
```

Comment:
Can't we have this string on a single line, it seems it fits within our max line length coding style, isn't it?
```suggestion
    warn(tr("'type' has been modified. This modification will be taken into account after saving and reloading the world."));
```